ugs-core/src/com/willwinder/universalgcodesender/GrblController.java

```diff
@@ -669,9 +669,9 @@ else if ((this.controllerStatus.getState().equals("Hold")
             if (zLocationMM > this.maxZLocationMM) {
                 maxZLocationMM = zLocationMM;
             }
-        }
 
-        dispatchStatusString(controllerStatus);
+            dispatchStatusString(controllerStatus);
```

Comment:
Was this change intentional?

Reply:
It was not my intention to commit it, but I did revert it in the next commit.